### ISSUE TITLE
Main menu is scrolled to the bottom on first reveal

### ISSUE THREAD

**Issue by mtschoen-unity** [Unity-Technologies/EditorVR#290]
You have to scroll up to see the console. Also, I believe that the original design was to cut the bottom menu button in half so that users know they can scroll it. We should do that, too so that the hidden workspaces are discoverable

**Comment by stella3d** [comment]
seems fixed ?  i see Console on first reveal, but we do still need to make the hidden ones discoverable

**Comment by mtschoen-unity** [comment]
Yes. The intent was to shrink/expand the menu height by half a button so you see one being cut in half.

I'll double-check that this is fixed.
